src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -147,6 +149,35 @@
                 });
         }, eZ.adminUiConfig.autosave.interval);
     }
+    const addClickWorkflowHandler = (event) => {
+        const applyButton = doc.querySelector('[name="ezplatform_content_forms_content_edit[workflow][apply]"]');
+
+        if (workflowEventListener) {
+            return;
+        }
+
+        applyButton.dataset.isFormValid = 0;
+        workflowEventListener = true;
+        applyButton.addEventListener('click', clickHandler, false);
+    }
+    const removeClickWorkflowHandler = (event) => {
+        const applyButton = doc.querySelector('[name="ezplatform_content_forms_content_edit[workflow][apply]"]');
+
+        if (!workflowEventListener) {
+            return;
+        }
+
+        workflowEventListener = false;
+        applyButton.removeEventListener('click', clickHandler, false);
+    }
+
+    buttonsMenuWorkflow.forEach((buttonMenuWorkflow) => {
```

Comment:
I think the event listener will depend only on last item from the array. I mean if the lest item has property validate it will be attached if not if will be removed, or am I missing something?

Reply:
addClickWorkflowHandler and removeClickWorkflowHandler  it is added only once for a given workflow button in the right menu (there can be several) and for "applyButton" event listener clickHandler is added once or removed, therefore there is a flag that checks whether it has already been added or not before adding/removing this EventListener